src/components/Modal/index.tsx

```diff
@@ -15,6 +15,7 @@ import ReactDOM from 'react-dom';
 
 // Import other components
 import waitMs from '../../helpers/waitMs';
```

Comment:
Could you move this import to the right place? I think this issue was before you started, but let's fix it